requirements.txt

```diff
@@ -1,2 +1,12 @@
+beautifulsoup4==4.5.3
```

Comment:
I've added a commit which adds a test_requirements.txt (packages just needed for testing) and script_requirements.txt (packages just needed for the scripts in bin) - hopefully that makes sense.